README.md

```diff
@@ -195,6 +195,16 @@ When retrieving an image URL (via `getMapUrl()`) with effects applied, an error
   const imageBlob2 = await layer.getMap(getMapParamsWithEffects, ApiType.PROCESSING);
 ```
 
+### Stitching images
+
+Services limit the size of the output image per request (5000px for OGC and 2500px for Processing API). If we need a bigger image, we can issue multiple requests and stitch the results together in a canvas. A utility method `getHugeMap` allows us to do that seamlessly.
```

Comment:
I think both OGC (WMS) and Processing API are limited to 2500px. Will talk live to give more info.

Reply:
Thank you, fixed!

Reply:
Nice, resolving.